sdk/response.js

```diff
@@ -84,10 +84,7 @@ class Response extends Api {
    * @return {boolean}
    */
   hasReturn() {
-    return this
-      .getServiceSchema(this._name, this._version)
-      .getActionSchema(this._actionName)
-      .hasReturn();
+    return this.return !== undefined;
```

Comment:
Indeed. Revising. The problem is that the CLI won't work at all for calling `request.newResponse()` since it has to go through this. I will discuss with @orestes further. Other option is to not let `getServiceSchema()` throw an exception, other is to catch it in the `Middleware`. I have to check the documentation and see what fits better.